test/storage/log_test.cpp

```diff
@@ -198,7 +186,8 @@ TEST_F(WriteAheadLoggingTests, LargeLogTest) {
   while (in.HasMore()) {
     storage::LogRecord *log_record = ReadNextRecord(&in);
     if (log_record->TxnBegin() == transaction::INITIAL_TXN_TIMESTAMP) {
-      // TODO(Tianyu): This is hacky, but it will be a pain to extract the initial transaction. The LargeTransactionTest
+      // TODO(Tianyu): This is hacky, but it will be a pain to extract the initial transaction. The
```

Comment:
huh, how did this reformat happen? Nothing important, just looks a bit goofy, but I'm surprised.